protonj2/src/main/java/org/apache/qpid/protonj2/logging/ProtonLogger.java

```diff
@@ -88,7 +88,7 @@
 
     public void error(String message);
 
-    public void erro(String message, Object value);
+    public void error(String message, Object value);
```

Comment:
public api change

Reply:
API changes are ok at this point as we are still in milestone releases and not an official 1.0